common/src/main/java/org/terracotta/angela/common/tcconfig/holders/TcConfigHolder.java

```diff
@@ -269,14 +269,6 @@ public void addServer(final int stripeIndex, final String hostname) {
       node.setAttribute("host", hostname);
       node.setAttribute("name", "Server" + stripeIndex + "-" + serverIndex);
 
-      Element node3 = tcConfigXml.createElement("tsa-port");
-      node3.appendChild(tcConfigXml.createTextNode("" + (90 + stripeIndex) + "1" + serverIndex));
-      node.appendChild(node3);
-
-      Element node4 = tcConfigXml.createElement("tsa-group-port");
-      node4.appendChild(tcConfigXml.createTextNode("" + (90 + stripeIndex) + "3" + serverIndex));
-      node.appendChild(node4);
-
```

Comment:
Bugfix: this code was generating port numbers not randomly and regardless of the default value used when initializing servers.

Now, this code does not generate any ports, which will make use of the port allocator.